rest/replication_api_test.go

```diff
@@ -1195,7 +1184,7 @@ func TestReplicationAPIWithAuthCredentials(t *testing.T) {
 	err = json.Unmarshal(response.BodyBytes(), &configResponse)
 	require.NoError(t, err, "Error un-marshalling replication response")
 	replication2Config.Remote = "http://bob:xxxxx@remote:4984/db"
-	//checkReplicationConfig(&replication2Config, &configResponse)
+	// checkReplicationConfig(&replication2Config, &configResponse)
```

Comment:
Not really related to this PR but is this actually needed for anything? Can maybe remove it whilst we're here?

Reply:
sure

```suggestion
```